lib/recommender_system/irt_recommender.rb

```diff
@@ -0,0 +1,28 @@
+module RecommenderSystem
+  class IrtRecommender < RecommenderSystem::Recommender
+    def get_probability
+      los = self.learning_objects
+      list = Hash.new
+      los.each do |lo|
+        irt_values = lo.irt_values.first
```

Comment:
Inak, ak by sa upravoval DB model, tak bacha, v ziadnom pripade neupravovat existujuce uz pushnute migracie (pozeram, ze boli take commity v tejto vetve), pretoze to uz niekto moze mat pullnute u seba a spustene - a druhykrat sa migracia, ktora uz prebehla nespusti (a to potom plati aj pre server). V takom pripade treba spravit novu migraciu, ktora upravi typy/vymaze tabulku alebo stlpce/vytvori nove, atd. a idealne aj rata s existujucimi datami (napr. v tomto pripade by sa prehodili hodnoty z tabulky `irt_values` do tabulky `learning_objects` - v migracii moze byt normalny ruby kod pracujuci s ActiveRecordom, ale kedze to este nebolo nasadene, zaobideme sa aj bez prekopirovania dat, len nech sa to naozaj robi novou migraciou).